test/unit/middleware/thunk-middleware.test.js

```diff
@@ -8,4 +8,15 @@ describe('Thunk middleware', function() {
 
     expect(spy).toHaveBeenCalledWith(action, repo)
   })
+
+
+  it('does not treat function action arguments as thunks when they are directly returned', function() {
+    let action = fn => fn
+    let spy = jest.fn()
+    let repo = new Microcosm()
+
+    repo.push(action, spy).onDone(result => {
+      expect(result).toEqual(spy)
+    })
```

Comment:
In case we ever make actions async, could you add an `expect.assertions(1)` to the top of this test?